capture_hdr_loop.py

```diff
@@ -9,60 +7,47 @@
 def _main():
     app = zivid.Application()
 
-    print("Connecting to the camera")
+    print("Connect to the camera")
     camera = app.connect_camera()
+    # camera = app.create_file_camera("C:\\Program Files\\Zivid\\data\\MiscObjects.zdf")
```

Comment:
I don't think adding this here makes sense because you can just look at this emulated camera point cloud, not actually simulate taking images.

Reply:
Ok, I just used it to test out the loop (without calling camera.capture, just to check logic of the loop. Then I thought, "hey that might be useful for others" :)